app/src/main/res/layout/progress_dialog_fragment.xml

```diff
@@ -14,38 +14,33 @@
   ~ along with this program. If not, see <https://www.gnu.org/licenses/>.
   -->
 
+
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@drawable/rounded_corners_9dp"
+    android:backgroundTint="@color/background"
     android:gravity="center"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:padding="32dp">
+
+    <ProgressBar
+        style="?android:attr/progressBarStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:indeterminateTint="@color/colorAccent" />
 
-    <LinearLayout
```

Comment:
This is actually needed. Around the progress wheel there is a rounded white background.

Reply:
I will revert, thanks.

Reply:
Nevermind it works. Can you explain here what is happening since the layout is match_parent? @dcampogiani

Reply:
I kept match parent because on master the root linear layout is using match parent.

I've removed the child LinearLayout with wrap_content values and moved the background attributes directly to the parent. So sizes should have no impact on the ProgressDialogFragment.

If you'd like to merge other changes and do further investigation on this one I can revert changes on this file. @marcouberti